docs/manual/common/concepts/CoreConcepts.md

```diff
@@ -10,7 +10,7 @@ The Lagom framework includes libraries and a development environment that suppor
     * [DC/OS](https://dcos.io/), an open-source, distributed operating system based on the Apache Mesos distributed systems kernel, created and maintained by Mesosphere, Inc.
     * Any cloud or on-premises deployment environment that supports private networking between hosts.
 
-Designing a microservices system that achieves high scalability and manifests resilience in the face of unexpected failures is extremely difficult. Without a framework such as Lagom, you would need to deal with all of the complex threading and concurrency issues inherent in highly distributed systems. By using Lagom as it was designed to be used, you can avoid many of these pitfalls and increase productivity at the same time. But, rather than throwing everything out and starting anew, Lagom allows you to adopt a [reactive architecture](https://info.lightbend.com/COLL-20XX-Reactive-Microservices-Architecture-RES-LP.html) within existing constraints. For example, you can create microservices that:
+Designing a microservices system that achieves high scalability and manifests resilience in the face of unexpected failures is extremely difficult. Without a framework such as Lagom, you would need to deal with all of the complex threading and concurrency issues inherent in highly distributed systems. By using Lagom as it was designed to be used, you can avoid many of these pitfalls and increase productivity at the same time. But, rather than throwing everything out and starting a new, Lagom allows you to adopt a [reactive architecture](https://info.lightbend.com/COLL-20XX-Reactive-Microservices-Architecture-RES-LP.html) within existing constraints. For example, you can create microservices that:
```

Comment:
Hi @qiuyangnie,

I think the usage of [`anew`](https://www.thefreedictionary.com/anew) in this sentence is correct. Was it confusing when you read it?